src/main/java/com/leggeart/legge/LawsuitService.java

```diff
@@ -0,0 +1,21 @@
+package com.leggeart.legge;
+
+import java.util.List;
+import java.util.Optional;
+
+
+/**
+ Lawsuit service interface.
+ @version 0.0.1
+ @author LC
+ */
+public interface LawsuitService {
```

Comment:
@Service
we are using Spring, annotations ad the top of every class is strongly recommended.